src/common/rollbar.config.js

```diff
@@ -50,10 +50,14 @@ function rollbarConfig(envServiceProvider, $provide) {
           // Parse the exception to get the stack
           stackFramesPromise = stacktrace.fromError(args[0], {offline: true});
         }else{
+          if (args[1] && (args[1].status === -1 || args[1].status === 401)) {
```

Comment:
Do we want to drop browser network errors?  Sometimes that is helpful when responding to tickets, just to know it was a connection issue for sure.  I haven't gotten those tickets in a while though, so I'm open to removing them.

Reply:
If you've found them useful, we can keep them. Spencer was complaining that there were too many errors in the missionhub-web project and a bunch were network errors. I figured it would be good to do the same thing here. There are a lot of them. I know the rollbar pricing plans are based on how many errors rollbar receives.

How have you used them? Looking up a specific person when they report an issue and see if some network issues are there? I could see that being useful.

Reply:
Yes, that's what I have used them for in the past, but haven't in a while.